### PR TITLE
SSD-123 - Add analysis by recent commit activity to overview

### PR DESCRIPTION
**Pull request by danjoneslf**

### CODE REVIEW

File: templates/components/content_blocks/repository_summary.html [hunk 1]
```diff
@@ -49,13 +49,13 @@ <h4 class="govuk-heading-s">Activity</h4>
               </dd>
             </div>
             {% endif %}
-            {% if repo.recentCommitDaysAgo %}
+            {% if repo.recentCommitDaysAgo is not none %}
```
Comment (0atman) [inline templates/components/content_blocks/repository_summary.html]: 1. This seems almost identical logic?
2. Do you mean `None`, capital N?

Comment (danjoneslf) [reply templates/components/content_blocks/repository_summary.html]: The reason for this is the case where the value is 0 (committed today) which is falsy. 
For some reason None is "none" in Jinja syntax

**Comment by danjoneslf** [comment]
Rebased to pick up the Dockerfile merge

File: templates/components/content_blocks/repository_summary.html (hunk 1 above)
Comment (0atman) [reply templates/components/content_blocks/repository_summary.html]: Huh. OK!

**Pull request merged**